ovis_bringup/launch/ovis_sleep.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+
+  <arg name="kinova_robotType" default="c2s6s000" />
+  <arg name="kinova_robotName" default="arm" />
+  <arg name="kinova_robotSerial" default="not_set" />
+  <arg name="use_jaco_v1_fingers" default="false" />
+  <arg name="feedback_publish_rate" default="0.1" />
+  <arg name="node_start_delay" default="5.0" />
```

Comment:
At first sight,  this line is the only change from the other launch. If I remember correctly, node_start_delay was experimentally shoehorned in during the RoboCup. Did it actually fix the launch issues or is this a dead file? 
If it works, why do we need two launch files?

Reply:
I think we can remove it, we don't currently use it.